lib/houston/client.rb

```diff
@@ -80,5 +80,16 @@ def unregistered_devices
     def devices
       unregistered_devices.collect{|device| device[:token]}
     end
+
+    def certificate_data
+      return unless ENV['APN_CERTIFICATE']
+      if File.exist?(ENV['APN_CERTIFICATE'])
+        File.read(ENV['APN_CERTIFICATE'])
+      else
+        # Fallback and return the certificate itself; if it isn't readable, the
+        #   env var is likely the whole string version of the cert's `.pem`.
+        ENV['APN_CERTIFICATE']
```

Comment:
I like that, though I'd suggest you update the readme as well if you decide to go that route.